packages/web-frontend/src/containers/OverlayDiv/components/ProjectPage/LegacyProjectCard.js

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
This file is just a copy paste of the old ProjectCard so doesn't really need review.